src/geff/interops/ctc.py

```diff
@@ -0,0 +1,217 @@
+import shutil
+from pathlib import Path
+
+import numpy as np
+import tifffile
+import typer
+import zarr
+from skimage.measure import regionprops
+from zarr.storage import StoreLike
+
+import geff
+from geff.metadata_schema import Axis, GeffMetadata
+from geff.write_arrays import write_arrays
+
+
+def ctc_tiffs_to_zarr(
+    ctc_path: Path,
+    output_store: StoreLike,
+    ctzyx: bool = False,
+    overwrite: bool = False,
+) -> None:
+    """
+    Convert a CTC file to a Zarr file.
+
+    Args:
+        ctc_path: The path to the CTC file.
+        output_store: The path to the Zarr file.
+        ctzyx: Expand data to make it (T, C, Z, Y, X) otherwise it's (T,) + Frame shape.
+        overwrite: Whether to overwrite the Zarr file if it already exists.
+    """
+    from dask.array.image import imread
```

Comment:
Can you somehow wrap this and put a warning telling people to install with [ctc]? See what Jan did for spatial graph in the __init__.py of the module for inspiration